packages/sdk/src/auth.ts

```diff
@@ -150,6 +156,7 @@ export class AuthManager {
         throw e;
       }
     }
+    if (!token) throw new Error('Missing token');
```

Comment:
typescript was yelling at me

Reply:
Yeah, `getTokenWithPopup` could be `undefined` so token is `string | undefined` if like 154 is the last executing line. You could probably push this up to there.